addon/data/selector-ui.js

```diff
@@ -159,6 +159,72 @@ const ui = (function () { // eslint-disable-line no-unused-vars
+        this.selectModeBackground.id = "pageshot-select-mode-background";
+        this.selectModeBackground.className = "pageshot-select-mode-background";
+        this.selectMode = document.createElement("div");
+        this.selectMode.className = "pageshot-select-mode";
+        this.selectMode.textContent = "Choose what to save";
+        this.selectModeBackground.appendChild(this.selectMode);
+        this.modes = document.createElement("div");
+        this.chooseRegionMode = document.createElement("button");
+        this.chooseRegionMode.className = "pageshot-select-mode-square-button";
+        this.chooseRegionMode.style.left = "60px";
+        let regionImg = document.createElement("img");
+        regionImg.src = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEsAAABLCAYAAAA4TnrqAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAB1WlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNS40LjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx0aWZmOkNvbXByZXNzaW9uPjE8L3RpZmY6Q29tcHJlc3Npb24+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgICAgIDx0aWZmOlBob3RvbWV0cmljSW50ZXJwcmV0YXRpb24+MjwvdGlmZjpQaG90b21ldHJpY0ludGVycHJldGF0aW9uPgogICAgICA8L3JkZjpEZXNjcmlwdGlvbj4KICAgPC9yZGY6UkRGPgo8L3g6eG1wbWV0YT4KAtiABQAABBRJREFUeAHtm79v00AUx79OSiA/aEMKUcpSoaogxMZOEexITJ26MVCJgalZKP9Ax7KhiKkL4k9AQkUs9B9AHUCIUqmhgaqlSUBtYnPn5C402JzvkjqJ/E6Kcn7+2u/88fM9+862Go2GAyqBCMQCqUjkEiBYGoFAsAiWBgENKUUWwdIgoCGlyCJYGgQ0pBRZBEuDgIaUIotgaRDQkI4JbblcxtraGra2ttBsNoU50v+xWAyFQgHz8/OYnZ2FJUYdlpeXsbm5GWk4fgefzWZRKpUg+6zt7W0/beTt+/v7qNVqHViRJ6IA4DgOZJ/VrV1ZWUEqleo2R2a5WCyiXq+fOF5fWPl8HplM5oQ4Sgu8c+8u/1q6FbQsCRAsiUJdIVhqRlJBsCQKdYVgqRlJBcGSKNQVgqVmJBUES6JQV3xvSlWbPt1o4tUnG7t1CzZ7AeBiEu6z0/M7caTPAA/XbVSPOm8G9Gqv/AbYEwcuJR3cvmyhdNe46apD811v7PHFBwcHx3y/LSDl9pNBnb06EbOAnZqNQ3d9y3e/7DvMz8uPwOockDJuvS+P/64wdldrAIwJijfjyCaAPHuM5MsZFlW8PJuLuxHXWurd/p1FFo/gJ+9t9s9O1NEIweIN53iuZYGCx/N2nl2WXsXULnzwS5H7bbYC2svFqdmMO/hUO4J4/xRmkX6Nrwnz1hrDMt7QvK3uloPyy50bn5/VWzH3UhB9VI8MAm8u/GbPBt6kb0JjWLxDH0QRfnkyCbsYR/Xjdw4evLFR++v2IIzGC78/WTYMuxhH1re6495H2SG3WPgdqWwYMqOhcGd8GQ5F60NuhPFlKLISZcMAZ0xkpQDSvkqEX8qGAbBSNgwASUgoGwoSQ/5P2VDjBFE2DAOWyEoavvoiFX4pGwbASdkwACQhGclsWGdj8LyEPeog/B4OYNTBOBu2xuAR+lj4oPzywDDOhnwsvMrGsh69bc22TJ6z3Ckw/szIx+V531I97swq9Gr/0Z7d4Xvkv4lRGim9MQlslIFjd0DLwu4vzp5NV7X+IPqW9mLf7BzUVMrBBJt+C7sYR9bre2P4fAB8OXTc+bwp9kYlv6anx1vzh+v34ycu0V7tX6uOu7/p8xaujFuID+DewRgWn3WeYXOGM1nvVl+94H3eTe3Xc95+vL2cjtW4gz+d5gz3XgmWxvkhWARLg4CGlCKLYGkQ0JD63josLS3B65MMjX2PtJR/BdZdfGFVKpVubeSXZZ81Ps5uval4EkgkEkgmk53vDRcWFpBOpz3FUTZySIuLi4jH453PfjkQ27axt7cH9ilwlPnIY+eAcrmcC4ob5TfSUkEVXwKyz/JV0ApJgGBJFOoKwVIzkgqCJVGoKwRLzUgqCJZEoa4QLDUjqSBYEoW68gekmoYeRy70ZgAAAABJRU5ErkJggg==";
```

Comment:
You should leave a comment in to indicate where the image source comes from, so it can be rebuilt.